omim2obo/main.py

```diff
@@ -139,8 +139,9 @@ def omim2obo(use_cache: bool = False):
     # Populate graph
     # - Non-OMIM triples
     graph.add((URIRef('http://purl.obolibrary.org/obo/mondo/omim.owl'), RDF.type, OWL.Ontology))
-    graph.add((URIRef('http://www.geneontology.org/formats/oboInOwl#hasSynonymType'), RDF.type, OWL.AnnotationProperty))
-    graph.add((URIRef('http://purl.obolibrary.org/obo/mondo#omim_included'), RDF.type, OWL.AnnotationProperty))
+    graph.add((URIRef(oboInOwl.hasSynonymType), RDF.type, OWL.AnnotationProperty))
+    graph.add((URIRef(MONDONS.omim_included), RDF.type, OWL.AnnotationProperty))
+    graph.add((URIRef(MONDONS.ABBREVIATION), RDF.type, OWL.AnnotationProperty))
```

Comment:
update to OMO abbreviation property

Reply:
Sounds good! I'll make that same change here.

Reply:
You know, this was my solution before to our recent problems with pronto's error.. I would also declare that these are sub-properties of `oboInOwl:hasSynonymType`. It wouldn't hurt to do that here as well, either in this PR or later.

Reply:
sure, whichever PR is best given how all these PRs/branches are being managed.